components/measurement/nettests/Dash.js

```diff
@@ -63,7 +63,8 @@ const getOptimalQualityForBitrate = (testKeys) => {
 const messages = defineMessages({
   dash: {
     id: 'Measurement.Metadata.Dash',
-    defaultMessage: 'See results of Dash Test on {date} in {country} and other measurements on OONI Explorer.'
+
```

Comment:
This shows up as 
![image](https://user-images.githubusercontent.com/700829/109030715-09674e00-7692-11eb-97c6-eac49c876710.png)

Reply:
Good call. I suspect similar grammar breakage will occur on the NDT test result as well. The cleanest thing to do would be to separate the `description` from the `title` and handle the differently in every page, but it's going to be quite a bit more work.

Is there some way to extract the actual video speed inside of the test result and display that in the description?